ui/app/src/hooks/useCurrencyFieldState.ts

```diff
@@ -8,6 +8,7 @@ const globalState = {
   toAmount: ref<string>("0"),
   priceImpact: ref<string | null>(null),
   providerFee: ref<string | null>(null),
+  slippage: ref<string>("0"),
```

Comment:
this is starting to hold more than the state for the currency field and probably could be renamed?

Reply:
So I removed some of this state in another PR as if we need to sync state I am thinking it might be better to do so with the store as our view model?